assessment/report_generator.go

```diff
@@ -189,3 +196,106 @@ func sourceColumnDefinitionToString(columnDefinition utils.SrcColumnDetails) str
 	}
 	return s
 }
+
+// TODO move calculation logic to assessment engine
+func calculateTableDbChangesAndImpact(srcTable utils.TableDetails, spTable utils.TableDetails) (string, string) {
+	changes := []string{}
+	impact := []string{}
+	if !strings.Contains(srcTable.Charset, "utf8") { // TODO add charset level comparisons - per source
+		changes = append(changes, "charset")
+		impact = append(impact, "storage increase")
+	}
+	if len(changes) == 0 {
+		changes = append(changes, "None")
```

Comment:
if len(impact) == 0 {
		impact = append(impact, "None")
	}
	
This also needs to be added

Reply:
added